vsts/pipelines/baseImages/php-fpm.yml

```diff
@@ -34,8 +25,8 @@ jobs:
 
 - job: Release_PhpRuntimeBaseImage
   dependsOn: 
-    - Build_Php_Fpm_Base
     - Build_Php_Fpm_Buster_Base
+    - Build_PHP_FPM_Bullseye_Base
```

Comment:
nit: can we change `FPM` to `Fpm`, or vice versa, just for consistency in this file 🙂

Reply:
Updated